static/src/js/components/GranuleResults/GranuleResultsItem.js

```diff
@@ -2,32 +2,179 @@
-  const temporal = granule.formatted_temporal
+const GranuleResultsItem = ({
+  collectionId,
+  granule,
+  isLast,
+  location,
+  waypointEnter,
+  onExcludeGranule,
+  onFocusedGranuleChange
+}) => {
+  const handleRemoveClick = () => {
+    const {
+      id,
+      is_cwic: isCwic
+    } = granule
+
+    let hashedId
+    if (isCwic) hashedId = murmurhash3(id).toString()
+    onExcludeGranule({
+      collectionId,
+      granuleId: hashedId || id
+    })
+  }
+
+  const handleClickGranuleDetails = (granuleId) => {
+    onFocusedGranuleChange(granuleId)
+  }
+
+  const {
+    browse_flag: browseFlag,
+    formatted_temporal: formattedTemporal,
+    id,
+    links,
+    online_access_flag: onlineAccessFlag,
+    producer_granule_id: producerGranuleId,
+    thumbnail: granuleThumbnail,
+    title: granuleTitle
+  } = granule
+
+  const title = producerGranuleId || granuleTitle
+  const temporal = formattedTemporal
   const timeStart = temporal[0]
   const timeEnd = temporal[1]
-  const thumbnail = granule.browse_flag ? granule.thumbnail : false
+  const thumbnail = browseFlag ? granuleThumbnail : false
+
+  const createDataLinks = () => {
+    // All 'http' data links that are not inherited
+    const httpDataLinks = links.filter((link) => {
+      const {
+        href,
+        inherited = false,
+        rel
+      } = link
+
+      return href.indexOf('http') !== -1 && rel.indexOf('/data#') !== -1 && inherited !== true
+    })
+
+    // Strip filenames from httpDataLinks
+    const filenames = httpDataLinks.map(link => link.href.substr(link.href.lastIndexOf('/') + 1).replace('.html', ''))
+
+    // Find any 'ftp' data links that are not interited that have filenames not already included with 'http' links
+    const ftpLinks = links.filter((link) => {
+      const {
+        href,
+        inherited = false,
+        rel
+      } = link
+
+      const filename = href.substr(href.lastIndexOf('/') + 1)
+
+      return href.indexOf('ftp://') !== -1 && rel.indexOf('/data#') !== -1 && inherited !== true && filenames.indexOf(filename) === -1
+    })
+
+    // Return http and ftp data links with a unique list of filenames, prefering http
+    return [
+      ...httpDataLinks,
+      ...ftpLinks
+    ]
+  }
+
+  const dataLinks = createDataLinks()
 
   return (
     <li className="granule-results-item">
       <div className="granule-results-item__header">
         <h3 className="granule-results-item__title">{title}</h3>
       </div>
       <div className="granule-results-item__body">
-        { thumbnail && (
-          <div className="granule-results-item__thumb">
-            <img src={thumbnail} height="85" width="85" alt={`Browse Image for ${title}`} />
-          </div>
-        )
+        {
+          thumbnail && (
+            <div className="granule-results-item__thumb">
+              <img src={thumbnail} height="85" width="85" alt={`Browse Image for ${title}`} />
```

Comment:
We use these dimensions when we request this data in the collection panel as well, I think it'd be valuable to define these dimensions in a config some place instead of hardcoding them.

Reply:
I'm all for this. Just wondering where we should put it. Any thoughts?

Reply:
We can hold off until we determine a configuration tool. Currently weighing options between using ENV vars and a JSON configuration file.